### ISSUE TITLE
Array of enums is broken after transformSchema

### ISSUE THREAD

**Issue by alek-wb** [yaacovCR/graphql-tools-fork#8]
Hello, we have a following schema:

```
enum EnumField {
  FirstValue
  SecondValue
  ThirdValue
}

type ExampleType {
  arrayEnumField: [EnumField!]!
}
```

`arrayEnumField` is a custom field resolver returning array of EnumField values. 

When querying the field after `transformSchema` we get an error: `[GraphQLError: Expected a value of type "EnumField" but received: "FirstValue"]`

It's similar to the error for plain enums that have been fixed in this fork, but array of enums is still broken.

Any chance it gets fixed anytime soon?

**Comment by yaacovCR** [comment]
Hope so, maybe over holiday.

**Comment by yaacovCR** [comment]
I think this is the line not handling lists:

https://github.com/yaacovCR/graphql-tools-fork/blob/master/src/stitching/checkResultAndHandleErrors.ts#L64

Logic of that section needs to be reworked, but fix seems straightforward.

**Comment by yaacovCR** [comment]
Should be fixed in v6.1.3.
